contrib/scripts/sensitive_strings.py

```diff
@@ -1,14 +1,12 @@
+import argparse
 import copy
 import cv2
-import hashlib
 import os
 import numpy as np
 from PIL import Image
 import sys
 import time
```

Comment:
Not a real problem, but I believe to be completely PEP8 compliant, the order of imports should be: 
1. Standard library imports
2. Third party imports
3. Local library imports
https://peps.python.org/pep-0008/#imports

Not a big deal, but this is what I have been following.

Reply:
Thanks, I didn't know about this part of the standard! Fixed!